test/ussd_public.test.js

```diff
@@ -482,6 +510,26 @@ describe("Hello Mama app", function() {
                         })
                         .run();
                 });
+                it("to state_optout_receiver", function() {
+                    return tester
+                        .setup.user.addr('082222')
+                        .inputs(
+                            {session_event: 'new'}  // dial in
+                            , '1'  // state_msisdn_permission - yes
+                            , '5'  // state_main_menu - stop receiving messages
+                            , '5'  // state_optout_reason - other
+                        )
+                        .check.interaction({
+                            state: 'state_optout_receiver',
+                            reply: [
+                                "Who would you like to stop receiving messages?",
+                                "1. Only me",
+                                "2. The Father",
```

Comment:
We need to add another state here that does not give option 2 when the role player is the father.